### PR TITLE
Indexer fixes

### PR DESCRIPTION
**Pull request by dzhelezov**
After https://github.com/Joystream/hydra/pull/23
This PR bring breaking changes to how the data is stored in the DB and require a full chain reindex:

- Gracefully handle websocket timeouts
- Remove extra quote escaping for JSON (by removing JSON serializer)
- Event name is always <section>.<method>
- Increase event index padding to 6

### CODE REVIEW

**Review by metmirr — APPROVED** [review]
~Looks good!~ I didn't see new commits.

File: substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts [hunk 1]
```diff
@@ -68,9 +65,9 @@ export class SubstrateEventEntity {
     _entity.index = q.indexInBlock;
     _entity.id = SubstrateEventEntity.formatId(_entity.blockNumber, _entity.index);
     _entity.method = q.event_record.event.method || 'NO_METHOD';
-    _entity.name = q.event_name;
-    _entity.phase = q.event_record.phase.toJSON();
     _entity.section = q.event_record.event.section || 'NO_SECTION';
+    _entity.name = `${_entity.section}.${_entity.method}`;
```
Comment (metmirr) [inline substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts:69]: Event handler name pattern was `section_method` and we do comparison by event name in order to call mapper.

Comment (dzhelezov) [reply substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts:69]: Actually it does not (and should not) affect the mappers, as it's just the raw events data. The mapping code itself should responsible for matching the handlers with the entity names.

Comment (metmirr) [reply substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts:69]: 👍

**Pull request merged**